app/views/team_mailer/team_registered_to_event.html.erb

```diff
@@ -1,6 +1,6 @@
 <h1><%= t('mailer.hello', firstname: @user.first_name) %></h1>
-<p><%= t('mailer.team_mailer.team_registered_to_event.registered_to_event_info_html',
+<p><%= t('.registered_to_event_info_html',
          team_link: link_to(@team.name, team_url(@team)),
          event_link: link_to(@event.name, event_url(@event))) %></p>
-<p><%= t('mailer.team_mailer.team_registered_to_event.have_fun') %></p>
+<p><%= t('mailer.have_fun') %></p>
```

Comment:
@conradlempert @tomkellog There were different `have_fun` localizations on purpose:

![image](https://user-images.githubusercontent.com/807665/35514605-01eb51d0-0507-11e8-9516-3594e2d2cab5.png)

This was caused by #307 requiring different “Have fun” messages for the different emails. Am I right that after your changes every email has the same “Have fun” message?

Reply:
Ja, ich habe hier eigenmächtig angenommen, dass der Unterschied zwischen "Viel Spaß beim Sport", "Viel Erfolg beim Spielen" und "Viel Erfolg beim Training" egal ist, und das bloß ungenau zwischen den verschiedenen Emails abgestimmt war. Wenn das doch wichtig ist, kann man die natürlich auch wieder reinnehmen

Reply:
Das wäre vielleicht eine Frage für unseren PO @svitlanakalancha … ich frag sie mal im unserem Koordinations-Slack-Chat.